scripts/workers/restart_rabbitmq_worker.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+
+#    List the process ids for all the running consumer or receiver processes.
```

Comment:
Lets make the comment in present imperative tense. like `Lists`, `Kills`, `Runs` and `Lists`.
Can we update the comments

```
Lists the process ids for all the running worker processes.
Kills the running worker processes.
Runs new processes for worker and lists their ids.
```